internal/run/lint_test.go

```diff
@@ -0,0 +1,163 @@
+package run
+
+import (
+	"fmt"
+	"github.com/golang/mock/gomock"
+	"github.com/stretchr/testify/suite"
+	"strings"
+	"testing"
+)
+
+type LintTestSuite struct {
+	suite.Suite
+	ctrl            *gomock.Controller
+	mockCmd         *Mockcmd
+	originalCommand func(string, ...string) cmd
+}
+
+func (suite *LintTestSuite) BeforeTest(_, _ string) {
+	suite.ctrl = gomock.NewController(suite.T())
+	suite.mockCmd = NewMockcmd(suite.ctrl)
+
+	suite.originalCommand = command
+	command = func(path string, args ...string) cmd { return suite.mockCmd }
+}
+
+func (suite *LintTestSuite) AfterTest(_, _ string) {
+	command = suite.originalCommand
+}
+
+func TestLintTestSuite(t *testing.T) {
+	suite.Run(t, new(LintTestSuite))
+}
+
+func (suite *LintTestSuite) TestPrepareAndExecute() {
+	defer suite.ctrl.Finish()
+
+	l := Lint{
+		Chart: "./epic/mychart",
+	}
+
+	command = func(path string, args ...string) cmd {
+		suite.Equal(helmBin, path)
+		suite.Equal([]string{"lint", "./epic/mychart"}, args)
+
+		return suite.mockCmd
+	}
+
+	suite.mockCmd.EXPECT().
+		Stdout(gomock.Any())
+	suite.mockCmd.EXPECT().
+		Stderr(gomock.Any())
+	suite.mockCmd.EXPECT().
+		Run().
+		Times(1)
+
+	cfg := Config{}
+	err := l.Prepare(cfg)
+	suite.Require().Nil(err)
+	l.Execute(cfg)
+}
+
+func (suite *LintTestSuite) TestPrepareRequiresChart() {
+	// These aren't really expected, but allowing them gives clearer test-failure messages
+	suite.mockCmd.EXPECT().Stdout(gomock.Any())
+	suite.mockCmd.EXPECT().Stderr(gomock.Any())
```

Comment:
These should have a `.AnyTimes()` at the end; the intention here is just "allow these calls," not "verify that these calls happened."